ambari-server/src/main/resources/stacks/HDP/2.5/role_command_order.json

```diff
@@ -21,7 +21,7 @@
   "_comment" : "Dependencies that are used when GLUSTERFS is not present in cluster",
   "optional_no_glusterfs": {
     "SPARK2_JOBHISTORYSERVER-START" : ["NAMENODE-START", "DATANODE-START"],
-    "SPARK2_THRIFTSERVER-START" : ["NAMENODE-START", "DATANODE-START", "HIVE_SERVER-START"]
+    "SPARK2_THRIFTSERVER-START" : ["NAMENODE-START", "DATANODE-START", "HIVE_SERVER-START", "HIVE_SERVER_INTERACTIVE-START"]
```

Comment:
is that cause problems if those components are on the same host?

Reply:
Can you please clarify the question?

Reply:
No, the problem may also happen if the components are on different hosts.  In that case this fix is not enough, a more complex fix in Spark2 ThriftServer start script is needed.

This is a simpler fix to get the "same host" scenario working.

Reply:
Is there any jira about the different host scenario?